pkg/kubelet/kubelet_volumes.go

```diff
@@ -58,6 +58,15 @@ func (kl *Kubelet) podVolumesExist(podUID types.UID) bool {
 		return true
 	}
 
+	volumePaths, err := kl.getMountedVolumePathListFromDisk(podUID)
```

Comment:
Add a comment describing that this is a best-effort check, and link to the github issue for the follow-up work.

Reply:
fixed